Example/Pods/Nimble/Sources/Nimble/Matchers/AllPass.swift

```diff
@@ -65,7 +65,7 @@ extension NMBObjCMatcher {
             
             var collectionIsUsable = true
             if let value = actualValue as? NSFastEnumeration {
-                let generator = NSFastEnumerationIterator(value)
+                var generator = NSFastEnumerationIterator(value)
```

Comment:
This change will get overwritten by a pod install. Update Nimble from the example project workspace's pod file and the run `pod update` to fix this.